sci-astronomy/fitspng/fitspng-1.4.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Utility to convert images in astronomical FITS to PNG format"
+DESCRIPTION="FITS format is a general purpose astronomical image format.
+FITS images stores measurements of a physical quantity - an array of the
+angular distribution of the photon events or the optical intensity.
+Fitspng does visualization of such array by converting them to the
+images in PNG format."
+HOMEPAGE="http://integral.physics.muni.cz/fitspng/"
+COPYRIGHT="2006 - 2019, F. Hroch,
+	Institute of Theoretical Physics and Astrophysics,
+	Masaryk University, Brno, Czech Republic."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="ftp://integral.physics.muni.cz/pub/fitspng/fitspng-$portVersion.tar.gz"
+CHECKSUM_SHA256="caa7fc2f0c90bbd14b9499a705d74bb07df9f6ecf53454b9614ec3b30459f89e"
+ARCHITECTURES="!x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fitspng${secondaryArchSuffix} = $portVersion
+	cmd:fitspng${secondaryArchSuffix} = $portVersion
```

Comment:
Don't use {} if not needed:
```diff
-	fitspng${secondaryArchSuffix} = $portVersion
-	cmd:fitspng${secondaryArchSuffix} = $portVersion
+	fitspng$secondaryArchSuffix = $portVersion
+	cmd:fitspng$commandSuffix = $portVersion
```
(for the cmd, check haikuports on how to use it combined with --omit-dirs)